mu4/domain/importexport/internal/musicxml/exportxml.cpp

```diff
@@ -6572,7 +6618,7 @@ void ExportMusicXml::writeMeasure(const Measure* const m,
     }
 
     // output attribute at start of measure: measure-style
-    measureStyle(_xml, _attr, m);
+    measureStyle(_xml, _attr, m, partIndex);
```

Comment:
So, previously the same measure style got written for each staff?

Reply:
Yes, that's right. It actually still is, because I haven't yet accounted for the "number" attribute that may or may not be present in the <measure-style> tag, but this is a WIP.

Reply:
Done.